docs/features/setting-storage.md

```diff
@@ -0,0 +1,116 @@
+# Setting Storage Extensions
+
+:heavy_exclamation_mark: There is also a version of [this document with code samples in VB.Net](./setting-storage.vb.md) :heavy_exclamation_mark: |
+----------------------------------------------------------------------------------------------------------------------------------------------- |
+
+SettingsStorageExtensions is a helper class that groups extension methods to facilitate the use of serialization, handling and storing of any content in a UWP application. This extension wraps the handling and storing of content (for example: custom content, key-values, files, etc.) in the same way as simple settings for a consistent interface.
+
+## Advantages:
+ - All methods of handling and storing content can be found in the same class and using a similar nomenclature. 
+ - Facilitate the serialization and deserialization of data content and make it transparent for the developers.
+ - Using methods that avoid problems in the serialization process (for example: big files that can take a long time to process).
+ - All methods automatically give default values back if the key you're trying to read doesn't exist.
+ - It makes working with byte arrays as easy as working with non-binary data (which can sometimes be complicated.)
+
+This helper contains extension methods for handling and storing content in:
```

Comment:
Need to explain the difference between these. The code examples below show no difference between the two.

Reply:
Between StorageFolder and ApplicationData?

Reply:
Yes, between StorageFolder and ApplicationData?